dbos/_client.py

```diff
@@ -158,8 +158,10 @@ async def enqueue_async(
         workflow_id = await asyncio.to_thread(self._enqueue, options, *args, **kwargs)
         return WorkflowHandleClientAsyncPolling[R](workflow_id, self._sys_db)
 
-    def retrieve_workflow(self, workflow_id: str) -> WorkflowHandle[R]:
-        status = get_workflow(self._sys_db, workflow_id, True)
+    def retrieve_workflow(
+        self, workflow_id: str, request: bool = True
```

Comment:
One thing we should do for V1 is completely get rid of this idea of storing HTTP requests in the database. It was a bad idea for a bunch of reasons.

Reply:
yeah I observe they are not serializable in the response most of the time

Reply:
OK so I am removing these filters from the PR